<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiSampleDialog</class>
 <widget class="QDialog" name="MultiSampleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi-sample analysis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This dialog is used to trigger a multi-sample analysis, e.g. for comparing a group of affected to unaffected samples. Enter processed sample names (e.g. GS110998_01). All samples have to be processes with the same processing system.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="samples_table">
     <column>
      <property name="text">
       <string>Sample</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Processing System</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="add_affected">
       <property name="text">
        <string>Add affected</string>
       </property>
       <property name="icon">
        <iconset resource="GSvar.qrc">
         <normaloff>:/Icons/Add_red.png</normaloff>:/Icons/Add_red.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_control">
       <property name="text">
        <string>Add control</string>
       </property>
       <property name="icon">
        <iconset resource="GSvar.qrc">
         <normaloff>:/Icons/Add_green.png</normaloff>:/Icons/Add_green.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_page_link">
       <property name="text">
        <string/>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="start_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start analysis</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="GSvar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>start_button</sender>
   <signal>clicked()</signal>
   <receiver>MultiSampleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
